src/routes/docs/tutorials/nuxt/step-7/+page.markdoc

```diff
@@ -0,0 +1,129 @@
+---
+layout: tutorial
+title: Show ideas on home page
+description: Add database queries using Appwrite in your Nuxt application.
+step: 7
+---
+
+# Show and add ideas {% #show-and-add-ideas %}
+
+Using the `useIdeas` composable we can now display the ideas on the page and add a form to submit new ideas.
+
+Overwrite the contents of `src/pages/index.vue` with the following:
+
+```vue
+<template>
+  <div>
+    <section v-if="user.isLoggedIn.value === true" class="u-margin-32">
+      <article class="box">
+        <h4 class="heading-level-4">Submit Idea</h4>
+
+        <form @submit.prevent="handleAddIdea" class="u-margin-block-start-16">
+          <ul class="form-list">
+            <li class="form-item">
+              <label class="label">Title</label>
+              <input
+                type="text"
+                placeholder="Title"
+                v-model="addIdeaData.title.value"
+              />
+            </li>
+            <li class="form-item">
+              <label class="label">Email</label>
+              <textarea
+                placeholder="Description"
+                v-model="addIdeaData.description.value"
+              />
+            </li>
+            <button class="button" aria-label="Submit idea" type="submit">
+              Submit
+            </button>
+          </ul>
+        </form>
+      </article>
+    </section>
+    <section v-else><p>Please login to submit an idea.</p></section>
+
+    <section class="u-margin-32">
+      <article class="card">
+        <h4 class="heading-level-4">Latest Ideas</h4>
+        <ul>
+          <li v-for="idea in ideas.current.value">
+            <div class="box">
+              <h5 class="heading-level-6">{{ idea.title }}</h5>
+              <p class="body-text-2">{{ idea.description }}</p>
+              <div
+                class="u-position-absolute u-inset-inline-end-8 u-inset-block-start-8"
```

Comment:
This code block is realllllyyy massive. Can it be more concise? Can we break it up?

Reply:
I broke it up in smaller pieces. Started with the form, moved on to the list and ended with the small styling tag.